javav2/example_code/secrets-manager/README.md

```diff
@@ -0,0 +1,78 @@
+# Secrets Manager code examples for the SDK for Java 2.x
+
+## Overview
+
+Shows how to use the AWS SDK for Java 2.x to work with AWS Secrets Manager.
+
+<!--custom.overview.start-->
+<!--custom.overview.end-->
+
+_Secrets Manager helps you to securely encrypt, store, and retrieve credentials for your databases and other services._
+
+## ⚠ Important
+
+* Running this code might result in charges to your AWS account. For more details, see [AWS Pricing](https://aws.amazon.com/pricing/) and [Free Tier](https://aws.amazon.com/free/).
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+
+### Prerequisites
+
+For prerequisites, see the [README](../../README.md#Prerequisites) in the `javav2` folder.
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+- [GetSecretValue](src/main/java/com/example/secrets/GetSecretValue.java#L6)
+
+
+<!--custom.examples.start-->
+<!--custom.examples.end-->
+
+## Run the examples
+
+### Instructions
+
```

Comment:
A java section is missing from `.tools/readmes/includes/run_instructions.jinja2`.
Please add that and re-run writeme, or add custom instructions here.

Reply:
I added a link to the main Java readme.